app/viewmodels/authorisedOfficials/AuthorisedOfficialsStatusHelper.scala

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.authorisedOfficials
+
+import models.UserAnswers
+import pages.QuestionPage
+import pages.addressLookup._
+import pages.authorisedOfficials._
+import viewmodels.StatusHelper
+
+object AuthorisedOfficialsStatusHelper extends StatusHelper {
+
+  private def journeyCommon(index:Int): Seq[QuestionPage[_]] = Seq(
+    AuthorisedOfficialsNamePage(index),
+    AuthorisedOfficialsDOBPage(index),
+    AuthorisedOfficialsPhoneNumberPage(index),
+    AuthorisedOfficialsPositionPage(index),
+    IsAuthorisedOfficialNinoPage(index),
+    AuthorisedOfficialAddressLookupPage(index),
+    IsAuthorisedOfficialPreviousAddressPage(index)
+  )
+
+  private def remainingJourneyPages(index:Int): Seq[QuestionPage[_]] = Seq(
+    AuthorisedOfficialsNinoPage(index),
+    AuthorisedOfficialsPassportPage(index),
+    AuthorisedOfficialPreviousAddressLookupPage(index)
+  )
+
+  private val authorisedOfficial1common: Seq[QuestionPage[_]] = journeyCommon(0) ++ Seq(IsAddAnotherAuthorisedOfficialPage)
+
+  private val authorisedOfficial2common: Seq[QuestionPage[_]] = journeyCommon(1)
+
+  private val allPages: Seq[QuestionPage[_]] =  {
+    authorisedOfficial1common ++ authorisedOfficial2common ++
+      remainingJourneyPages(0) ++ remainingJourneyPages(1)
+  }
+
+  private val f1 = (isNino:Boolean) => {
```

Comment:
could you please give the functions and variables clearer names (eg 'addProofOfIdentityPage' instead of 'f1'), and add spaces to make the document more scalastyle-y ('isNino: Boolean' instead of 'isNino:Boolean')? thank you!